types/react/index.d.ts

```diff
@@ -2789,7 +2789,7 @@ declare namespace React {
         name?: string;
         required?: boolean;
         size?: number;
-
+        value?: string | string[] | number;
```

Comment:
ah sorry, I see it's not my pull request, typed too fast. I missed this one.

This definition looks correct to me.